src/rhelocator/update_images/azure.py

```diff
@@ -229,3 +231,81 @@ def get_images() -> list[dict[str, str]]:
                     results.append(result)
 
     return results
+
+
+def parse_image_version(image_version: str) -> dict[str, str]:
+    """Parse an AWS image name and return extra data about the image.
+
+    Regex101: https://regex101.com/r/yHB9jJ/1
```

Comment:
Please give me a sanity check for this regex. I'm not 100% sure if all RHEL images follow this precise formula but it seems to be consistent within the images we currently retrieve.

Reply:
Haha, nicely done here. It looks like you're leaving off the optional last two digits after the year. Hopefully that's what you intended. 😉

Reply:
Haha yeah to be honest I haven't found a real use for the optional digits, but I guess we can revisit this at any time.